test_scripts/Polices/External_UCS/002_ATF_N_TC_LPT_Creation_PreloadedPT_with_external_consent_status_groups_struct.lua

```diff
@@ -0,0 +1,110 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT with "external_consent_status_groups" struct
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable with "external_consent_status_groups:
+-- [<functional_grouping>: <Boolean>]" -> of "device_data" -> "<device identifier>"
+-- -> "user_consent_records" -> "<app id>" section
+-- SDL must:
+-- a. consider such PreloadedPT is invalid
+-- b. log corresponding error internally
+-- c. shut SDL down
+--
+-- Preconditions:
+-- 1. Stop SDL
```

Comment:
@dboltovskyi, will be good to specify that SDL is built with External_Proprietary flag.